en/identity-server/next/docs/guides/users/manage-users.md

```diff
@@ -13,24 +13,34 @@ There are three ways to onboard a user:
 
 - An administrator can onboard users from the WSO2 Identity Server Console.
 
-Let's look at how administrators can onboard users from the WSO2 Identity Server Console.
+Let's look at how administrators can onboard users from the {{ product_name }} Console.
 
-1. On the WSO2 Identity Server Console, go to **User Management** > **Users**.
-2. Click  **Add User** and provide the following details:
+1. On the {{ product_name }} Console, go to **User Management** > **Users**.
+2. Click **Add User** and provide the following details:
 
-    ![Add user]({{base_path}}/assets/img/guides/users/add-user-form.png){: width="600" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}
+    ![Add user]({{base_path}}/assets/img/guides/users/add-user-form.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
 
     !!! note
+           - A username is always unique to the organization and you can't change the username once it is created.
+           - Instead of using a username to login, you can [configure email address as the username]({{base_path}}/guides/users/attributes/enable-email-as-username/). Then, you will be asked to enter the email address when logging in.
+           - The user can change the password set by the administrator from the [My Account Portal]({{base_path}}/guides/user-self-service/customer-self-service-portal/).
+
+3. You can either request the user to set the password or set one on the user's behalf.
+
+    - **Invite user to set their own password:**
+
+        - **Invite via email:** If this option is selected, an email with a confirmation link will be sent to the provided email address. The user can use this link to set up a new password.
```

Comment:
Since IS doesn't enable this by default, need to add a note how to enable this option

Reply:
Fixed and updated the PR